specification/attributes/amortization_handling.md

```diff
@@ -0,0 +1,34 @@
+# Amortization Handling
+
+Amortization is a process used to break down and spread purchase costs over the period of time or term of use. 
+
+Amortization is primarily used in purchasable commitment-based discounts. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on resource usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate significant purchase charges to the appropriate audience in support of their cost allocation efforts. 
+
+## Attribute ID
+
+AmortizationHandling
+
+## Attribute name
+
+Amortization Handling
+
+## Description
+
+Rules and requirements defining how amortization is shown in the billing data.
+
+## Requirements
+
+* Every charge MUST include all applicable discounts. (e.g., negotiated and commitment-based discounts)
```

Comment:
May negotiated costs imply a purchase, which is then amortized?

In my mind, but maybe I am not educated enough, costs are negotiated with providers based on a financial commitment, without any purchase.

Indeed, credits can be negotiated. To my knowledge, credits are not amortized.

Reply:
Negotiated discounts can also be discounts applied to an agreement for example retail price minus negotiated discount applies to on demand charges and is not necessarily an amortized cost. So do negotiated discounts need to be called out in amortization handling?